CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * `operator.size`, has been deprecated. If you were using this function, please transition to `operator.hilbert.size`. [#985](https://github.com/netket/netket/pull/985)
 
 ### Internal Changes
+* The `DenseSymm` layer checks if the provided `symmetries` is a `HashableArray`. If not, it tries to make it a `HashableArray`. [#989](https://github.com/netket/netket/pull/989)
```

Comment:
I'd reword this:

Is this an internal change? Our documentation did not really specify whether `DenseSymm` should accept `HashableArray` or not, but it seems to be more of a bug fix (or, technically, a new feature). @PhilipVinc 

Also, I'd reword this a bit (leaving out the internal workings of the constructor):
```suggestion
* The `DenseSymm` layer now also accepts objects of type `HashableArray` as `symmetries` argument. [#989](https://github.com/netket/netket/pull/989)
```

Reply:
You're right, it's a bug fix.